libqtile/window.py

```diff
@@ -1403,9 +1435,9 @@ def _is_in_window(self, x, y, window):
         return (window.edges[0] <= x <= window.edges[2] and
                 window.edges[1] <= y <= window.edges[3])
 
-    def cmd_set_position(self, dx, dy):
+    def cmd_set_position(self, dx, dy, dz=0):
```

Comment:
Should be None by default, not 0 (see [previous comment](https://github.com/qtile/qtile/pull/1870/commits/af47ffeb4273da1c0d8f9f6ed5982e22ffc1db4e#r493904171)).

Reply:
I get it. It is now `None`.